docs/OWNERS

```diff
@@ -1,9 +1,13 @@
-reviewers:
-  - brendandburns
-  - smarterclayton
-  - thockin
-approvers:
-  - brendandburns
-  - smarterclayton
-  - thockin
-  - pwittrock
\ No newline at end of file
+".*":
+	reviewers:
+	  - brendandburns
+	  - smarterclayton
+	  - thockin
+	approvers:
+	  - brendandburns
+	  - smarterclayton
+	  - thockin
+	  - pwittrock
+"^(getting-started-guides|admin|user-guide|devel|design|proposals)":
```

Comment:
Many of these names are no longer needed, can we remove them?

If this is to match subdirs, should i tinclude the trailing/ ?